neurots/generate/algorithms/basicgrower.py

```diff
@@ -51,19 +51,20 @@ def initialize(self):
 
         return stop, num_sec
 
-    def bifurcate(self, current_section):
+    def bifurcate(self, current_section, pia_direction=None):
         """When the section bifurcates two new sections are created.
 
         This method computes from the current state the data required for the
         generation of two new sections and returns the corresponding dictionaries.
 
         Args:
             current_section (neurots.generate.section.SectionGrowerPath): The current section.
+            pia_direction (numpy.ndarray): Direction of the pia if different from `[0, 1, 0]`.
```

Comment:
It can also be a simple Python list, right?

Reply:
yes, I can add in the type